app/Http/Middleware/Locale.php

```diff
@@ -13,10 +16,45 @@ class Locale
      * @param  \Closure  $next
      * @return mixed
      */
+    protected $usermeta;
+
+    public function __construct(UsermetaRepository $usermeta) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Middleware/Locale.php</br>**Line**: 21</br>Opening brace should be on a new line